Q/Qhull/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Qhull"
+version = v"2019.1"
+
+# Collection of sources required to build
+sources = [
+    GitSource(
+        "https://github.com/qhull/qhull",           # project URL
+        "b94cff0799f36068dee5d800d5cef3b9b911c400"  # commit hash corresponding to the version
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# initial setup
+cd $WORKSPACE/srcdir/qhull
```

Comment:
The safe thing is to always put a * at the end.

Reply:
Makes sense, the script was failing to find `qhull-*` earlier, so I wasn't sure what to put there.  Just fixed it (I think).

Reply:
Huh, this is interesting.  As of the last commit, 

```
]  ---> cd $WORKSPACE/srcdir/qhull*
[03:57:03] /bin/bash: line 33: cd: /workspace/srcdir/qhull*: No such file or directory
```

is where the builder errors.  I don't know why?

Reply:
I think it is Qhull

Reply:
Sorry about this. Maybe revert back to the ArchiveSource for now?